src/NServiceBus.Core/obsoletes.cs

```diff
@@ -854,6 +855,54 @@ public void Dispose()
             throw new NotImplementedException();
         }
     }
+
+    public partial class MessageHandlerRegistry
+    {
+        [ObsoleteEx(ReplacementTypeOrMember = "MessageHandlerRegistry.GetHandlersFor(Type messageType)", RemoveInVersion = "7", TreatAsErrorFromVersion = "6")]
+        public IEnumerable<Type> GetHandlerTypes(Type messageType)
+        {
+            throw new NotImplementedException();
+        }
+
+        [ObsoleteEx(ReplacementTypeOrMember = "MessageHandler.Invoke(object message, object context)", RemoveInVersion = "7", TreatAsErrorFromVersion = "6")]
+        public void InvokeHandle(object handler, object message)
+        {
+            throw new NotImplementedException();
+        }
+
+        [ObsoleteEx(ReplacementTypeOrMember = "MessageHandler.Invoke(object message, object context)", RemoveInVersion = "7", TreatAsErrorFromVersion = "6")]
+        public void InvokeTimeout(object handler, object state)
+        {
+            throw new NotImplementedException();
+        }
+
+        [ObsoleteEx(ReplacementTypeOrMember = "MessageHandlerRegistry.RegisterHandler(Type handlerType)", RemoveInVersion = "7", TreatAsErrorFromVersion = "6")]
```

Comment:
@SimonCropp how is `ReplacementTypeOrMember ` supposed to be used? (should only the member be specified in this case `RegisterHandler`)